lib/inventory_refresh/save_collection/saver/sql_helper_upsert.rb

```diff
@@ -107,10 +108,12 @@ def insert_query_on_conflict_update(all_attribute_keys, mode, ignore_cols, colum
       end
 
       def insert_query_on_conflict_update_mode(mode, version_attribute, column_name)
-        if mode == :full
+        case mode
```

Comment:
Same here...Minor, but we really should have an else clause with a `raise` (not for this PR)